Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -39,6 +39,7 @@ class TextureCacheBase
     bool is_efb_copy;
     bool is_custom_tex;
     bool may_have_overlapping_textures = true;
+    bool is_invalidated = false;  // used only for tmem cache emulation
```

Comment:
Maybe invert this to `is_valid`

I'm not sure. Or something more descriptive... like `tmem_only`